lib/src/syn/v2/parser/test/stmt.rs

```diff
@@ -315,7 +315,8 @@ fn parse_define_table() {
 				expiry: std::time::Duration::from_secs(1)
 			}),
 			comment: None,
-			table_type: TableType::Normal,
+			// TODO (2.0.0) (RaphaelDarley) : Change default to TableType::Normal
```

Comment:
We should have a deprecation notice if this is exposed so users can prepare an upgrade without downtime.

So this functionality needs to be provided in parallel really.